wolox-books/src/app/screens/auth/auth.component.scss

```diff
@@ -0,0 +1,16 @@
+@import './src/scss/colors';
+
+.navbar {
+  border-top: 5px solid $blue;
+  width: 100%;
+  box-shadow: 0 10px 6px -6px $grey;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 15%;
```

Comment:
don't use percentage paddings. the reason behind this is to have a better knowledge of how the view is going to be displayed in every screen.